### PR TITLE
Update English Release Notes

### PR DESCRIPTION
**Pull request by 4ioskd**
I partially translated the page of Japanese Release Notes( https://misskey-hub.net/docs/releases.html ) into English one and added notes of Misskey v12.110.1 and v12.110.0 to the page of English one.

### CODE REVIEW

File: src/en/docs/releases.md [hunk 1]
```diff
@@ -5,6 +5,64 @@ Notes for indivudual misskey releases.
 Information updates may not be translated yet. For the latest information see [GitHub](https://github.com/misskey-dev/misskey/blob/master/CHANGELOG.md).
 :::
 
+## 12.111.1（2022/06/13）
+
+### Bugfixes
+- some fixes of multiple notification read @tamaina
+- some GenerateVideoThumbnail failed @Johann150
+- Client: fix problem with widget information not being saved in Deck mode @syuilo
+- Client: edit screen appears when tring to open gallery posts @futchitwo
```
Comment (Johann150) [inline src/en/docs/releases.md:14]:
```suggestion
- Client: edit screen appears when trying to open gallery posts @futchitwo
```

File: src/en/docs/releases.md [hunk 2]
```diff
@@ -5,6 +5,64 @@ Notes for indivudual misskey releases.
...
+- Supports Unicode Emoji 14.0 @mei23
+- Push notifications for multiple accounts #7667 @tamaina
+- Set click and `action` for Push notifications #7667 @tamaina
+- Option to discard original image and only keep `WebPublic` one when uploading it to Drive @tamaina
+- Server: always remove completed tasks of job queue @Johann150
+- Client: allow cropping of images in avatar settings @syuilo
+- Client: make emoji stand out more on reaction button @Johann150
+- Client: display URL of QR code for TOTP registration @tamaina
+- Client: render quote renote CWs as MFM @pixeldesu
+- API: `notifications/read` also accepted in arrays #7667 @tamaina
```
Comment (Johann150) [inline src/en/docs/releases.md:31]:
```suggestion
- API: `notifications/read` also accepts in arrays #7667 @tamaina
```

File: src/en/docs/releases.md [hunk 3]
```diff
@@ -5,6 +5,64 @@ Notes for indivudual misskey releases.
...
+- API: if the query meets the `username` condition, `username` is also `LIKE` searched when searching users @tamaina
+- MFM: Allow speed changes in all animated MFMs @Johann150
+- The theme color is now better validated. @Johann150<br>
+  Your own theme color may be unset if it was in an invalid format.<br>
+  Admins should check their instance settings if in doubt.
+- Perform port diagnosis at startup only when Listen fails @mei23
+- Rate limiting is now also usable for non-authenticated users. @Johann150 @mei23<br>
```
Comment (Johann150) [inline src/en/docs/releases.md:38]: are these `<br>` s here on purpose? I don't think they are necessary.
```suggestion
- The theme color is now better validated. @Johann150
  Your own theme color may be unset if it was in an invalid format.
  Admins should check their instance settings if in doubt.
- Perform port diagnosis at startup only when Listen fails @mei23
- Rate limiting is now also usable for non-authenticated users. @Johann150 @mei23
```

**Review by Johann150 — CHANGES_REQUESTED** [review]

File: src/en/docs/releases.md (hunk 1 above)
Comment (4ioskd) [reply src/en/docs/releases.md:14]: Oops, Thank you for checking!

File: src/en/docs/releases.md (hunk 2 above)
Comment (4ioskd) [reply src/en/docs/releases.md:31]: Thank you for checking!

File: src/en/docs/releases.md (hunk 3 above)
Comment (4ioskd) [reply src/en/docs/releases.md:38]: hmm...ok. remove <br> s .

**Review by Johann150 — APPROVED** [review]

**Comment by 4ioskd** [comment]
Thank you for merging🙏🏻

**Pull request merged**